Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -553,7 +553,7 @@
   description: uplink-exploding-syndicate-bomb-desc
   productEntity: SyndicateBomb
   cost:
-    Telecrystal: 52 # goob
+    Telecrystal: 52 # goob 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)